CDCgov/prime-reportstream
# Create dashboard showing cost per message in production #10546

This PR introduces a GitHub Action that will run daily and populate a new `azure_costs` table with daily Azure costs for of Production resource group for the period of last 180 days. The table will be used for reporting in Metabase.

Test Steps:
1. Wait until trigger
2. Inspect the database and ensure `azure_costs` table is present
3. Ensure 180 rows are present with cost data.

## Changes
- A new GitHub Action (cron schedule)

## Checklist

### Testing

### Process

## Linked Issues
https://github.com/CDCgov/prime-reportstream/issues/10546

## To Be Done
Create a Dashboard in Metabase
